static/app/components/events/eventReplay/index.tsx

```diff
@@ -71,21 +78,39 @@ function EventReplayContent({
     <ReplaySectionMinHeight>
       <ErrorBoundary mini>
         <ReactLazyLoad debounce={50} height={448} offset={0} once>
-          <LazyLoad
-            component={replayPreview}
-            replaySlug={replayId}
-            orgSlug={organization.slug}
-            eventTimestampMs={eventTimestampMs}
-            buttonProps={{
-              analyticsEventKey: 'issue_details.open_replay_details_clicked',
-              analyticsEventName: 'Issue Details: Open Replay Details Clicked',
-              analyticsParams: {
-                ...getAnalyticsDataForEvent(event),
-                ...getAnalyticsDataForGroup(group),
-                organization,
-              },
-            }}
-          />
+          {!hasReplayClipFeature ? (
```

Comment:
nit: instead of negating the condition, just flip the order of the ternary return values:

```
- !hasThing ? <NoThing /> : <WithThing />
+ hasThing ? <WithThing /> : <NoThing />
```

Reply:
and maybe the condition can only wrap the component value, to make it clear that everything else is the same:
```
const asyncComponent = hasReplayClipFeature ? replayClipPreview : replayPreview;

<LazyLoad
  component={asyncComponent}
  ...
```